### DISCUSSION TITLE
[干货]如何写好你的绳网帖子

### DISCUSSION THREAD

**Discussion by inkluv** [share121/inter-knot#1092] [其他]
### 有很多新绳匠不怎么会用 _绳网_ 的 **Markdown** 格式来写好帖子！

这篇帖子给一些新手绳匠一些帮助，让你的帖子**不再单一**！
_懂的也不要嘲讽，谁不是这么过来的呢_

**如果帮到你，记得在评论区实践一下哦！**

在编写帖子时，看到输入框左上的第二个选项“Preview”了吗？
点击它，可以预览你 **帅气的帖子！**

看到右上方的一些按钮了吗？
我们按顺序讲一讲！

**首先是Heading！**

我们可以把Heading看成我们的 **标题！**
### 就像这样！
可以让你的文字变大，起突出的作用~

**然后是Bold！**

可以加粗你的字体！**就像这样！**
也可以突出强调，但是没有Heading那么明显~

**Italic可以让你的字 _变成斜的_ ！**  _记得两边不能紧挨着字符哦_ _不然会变成这样_awa

**Quote就是引用，比如：**

> ”有的绳匠在冒泡，但已经潜水了。“——法厄同

**Code是给编程大佬用的！！**

```
#include <iostream>
using namespace std;
int main(){
    return 0;
}
```

**Link可以引入链接！** _不要搞奇怪的东西哦~_

[点我，如果你是柚子厨](https://ciallo.cc/)

1. 这是
2. 数字
3. LIST

- 这是
- 无序LIST

- [ ] 这是目标LIST！
- [ ] 你完成了吗~
- [x] 在左边的方括号里面输入x，完成目标！ 

[这是附带文件.txt](https://github.com/user-attachments/files/16569916/default.txt)

@share121 这是提及某个人/大佬！（抱歉打扰awa

#1070  可以打开其他的帖子！井号+编号！
（这是我的第一篇帖子）

后面两个不太懂，写帖子好像没啥用

### 希望这篇帖子可以帮到你，成为传奇绳匠！

8.11编： 适当改动了内容~  _进阶教程_  ：#999 ←点这个

**Comment by inkluv** [comment]
更正：在绳网中，Code段无法一键复制

**Comment by inkluv** [comment]
有提议/更正，在这里说哦！谢谢！

**Comment by match-fire** [comment]
也可以[看这个](https://www.jianshu.com/p/82e730892d42#fn1)

**Reply by inkluv** [reply]
是的，绳网的markdown系统与正常的有所区分，当然想学习还得看你这个

**Comment by inkluv** [comment]
进一步学习Markdown请看楼上

**Comment by GanlinSama** [comment]
谢谢_大佬_
非常**实用**

**Comment by inkluv** [comment]
哦对了，使用斜体字时，要用空格与其他字隔开 _才有作用哦~_

**Comment by lllliufeng** [comment]
> [!TIP]
> 可在[这里](https://docs.github.com/zh/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)查看较为完善的语法~

**Comment by inkluv** [comment]
如果需要更详细的教程，请看楼上↑↑↑↑
除了数学公式、Married 图标，其他都支持 ——来自绳网作者

**Comment by fairy-B** [comment]
哇，好经典的C++模版！！
QWQ

**Comment by inkluv** [comment]
哈哈哈，只会CPP了（AwA）

**Comment by aminoc12138** [comment]
谢谢大佬，非常实用！

**Reply by inkluv** [reply]
悄悄告诉你：评论也能用哦

**Comment by inkluv** [comment]
回复楼上：其实这些都是Markdown基础，去B站大学或者CSDN都能学会的！OvO

**Comment by wugan-bot156** [comment]
又是羡慕编程大佬的一天Orz，可惜我是学化学的，也没有能力同时搞别的东西。/(ㄒoㄒ)/~~

**Comment by SixRadium** [comment]
**让我试试新学的东西！** `#abcdef` *哇，针不戳* 🥇 
## 大大大

**Reply by inkluv** [reply]
忘讲颜色了（

**Comment by Ellen666design** [comment]
感谢，劳斯，劳斯真好

**Comment by inkluv** [comment]
偷偷说一句：评论也可以使用这些功能哦~

**Comment by sekiroissekiko** [comment]
感谢帖主！赞美帖主！（

**Reply by inkluv** [reply]
AwA

**Comment by user1-cloud** [comment]
#999 (进阶教程）希望这个也能帮到各位

**Comment by inkluv** [comment]
我只演示了部分功能，需要进阶教程看楼上哦~
### 谢谢各位大佬补充~

**Comment by lllliufeng** [comment]
补一个上下标~

L<sup>A </sup>T<sub>E </sub>X
L\<sup>A\<\/sup>T\<sub>E <\/sub>X

**Reply by inkluv** [reply]
受教了OwO

**Comment by inkluv** [comment]
+3

**Comment by keerdun** [comment]
### 好实用的干货，感谢前辈指导！

**Comment by Subarasii-H** [comment]
多谢大佬

**Comment by fzdqz** [comment]
学费了|•'-'•)و✧

**Comment by RaInWH** [comment]
谢谢大佬，非常实用
